bedrock/firefox/templates/firefox/all-unified.html

```diff
@@ -146,6 +146,10 @@ <h1 class="c-intro-heading">{{ ftl('firefox-all-choose-which-firefox') }}</h1>
             </tr>
           </table>
 
+          <p class="c-linux-debian">
```

Comment:
Spotted an issue: I don't see this string on initial page load if the page detects `linux` as my platform. It only appears after I change platform using the select box?